build.sbt

```diff
@@ -149,8 +147,12 @@ lazy val test: Project = project
     libraryDependencies ++= Seq(
       "org.scalacheck" %% "scalacheck" % scalacheckVersion % Test,
       "org.typelevel" %% "cats-core" % catsVersion % Test
+    ),
+    protobufRunProtoc in ProtobufConfig := (args =>
```

Comment:
Moving the test `.proto`s to `test` to keep the project structure simple.

Reply:
fine with me, I couldn't get it to work locally that way :/

Reply:
Probably because of the `protobufVersion` variable which points to a version not available in `protoc-jar` yet.